src/actions/graph.js

```diff
@@ -27,6 +28,45 @@ export function updateGraph(graph) {
   };
 }
 
+export const TOGGLE_DISPLAY_LARGE_GRAPH = 'TOGGLE_DISPLAY_LARGE_GRAPH';
+
+/**
+ * resets the disaplyLargeGraph setting to enable large graphs to load
```

Comment:
I think best to stick to the standard used throughout here and capitalise the first letter of all comments / end with a full stop (if needed)

Reply:
Sure, thanks for picking up on the capitalization, however in looking through the codebase I don't see any comments ending on with the full stop?

Reply:
Ah that's fine on the full stop, I just meant where it makes sense (usually where there are multiple lines in a jsdoc)

Reply:
Thanks! I have updated the comment with the capitalization!